.github/workflows/cicd.yml

```diff
@@ -11,7 +11,7 @@ on:
   workflow_dispatch:
 
 env:
-  GOVER: 1.18.4
+  GOVER: 1.19.5
```

Comment:
Hi @steiler 
can you revert this for now. We will move to 1.20 when 1.21 will come out

Reply:
then our tests fail ... take a look at this:
https://github.com/srl-labs/containerlab/actions/runs/4085203521/jobs/7042866028
happened to me also in the other PR

Reply:
then we have to pin staticcheck to `v0.4.0` in the install command to keep 1.18 requirement